elements/pfe-toast/src/pfe-toast.scss

```
@@ -0,0 +1,118 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: toast;
+
+:host {
+  // local variables
+  --pfe-toast--Top: 50px;
+  --pfe-toast--MaxWidth: 500px;
+  --pfe-toast--MinWidth: 250px;
+  --pfe-toast--Right--offset: 50px; // needed with box-shadow
+  --pfe-toast--Right: calc(-1 * (#{pfe-local(MaxWidth)} + #{pfe-local(Right--offset)}));
+  --pfe-toast__container--BackgroundColor: #{pfe-color(surface--lightest)};
+  // --pfe-toast__container--BoxShadow: pfe-var(box-shadow--lg);
+  --pfe-toast__container--BoxShadow: rgba(3, 3, 3, 0.13) 0px 3px 7px 3px, rgba(3, 3, 3, 0.12) 0px 11px 24px 16px;
+  --pfe-toast__container--Color: #{pfe-color(text)};
+  --pfe-toast__container--Padding: 1rem;
+  --pfe-toast__close--svg--Fill: #{pfe-color(feedback--default--darkest)};
+  --pfe-toast__close--svg--Fill--hover: #151515;
+  --pfe-toast__close--svg--Height: 12px;
+  --pfe-toast__close--svg--Width: 12px;
+  --pfe-toast__close--PaddingTop: 0;
+  --pfe-toast__close--PaddingRight: 0;
+  --pfe-toast__close--PaddingBottom: 0;
+  --pfe-toast__close--PaddingLeft: 0;
+  --pfe-toast__close--Padding: #{pfe-local($cssvar: PaddingTop, $region: close)} #{pfe-local($cssvar: PaddingRight, $region: close)} #{pfe-local($cssvar: PaddingBottom, $region: close)} #{pfe-local($cssvar: PaddingLeft, $region: close)};
+  
+  // flex parent props
+  align-items: stretch;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  
+  // animation
+  position: absolute;
+  right: #{pfe-local(Right)};
+  top: #{pfe-local(Top)};
+  //transition: right #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+  transition: right 0.5s ease-in-out;
+  will-change: right;
+  
+  max-width: #{pfe-local(MaxWidth)};
+  min-width: #{pfe-local(MinWidth)};
+
+  @media only screen and (max-width: $pfe-global--grid-breakpoint--sm) {
+		max-width: calc(#{pfe-local(MaxWidth)} / 2);
+    min-width: calc(#{pfe-local(MinWidth)} / 2);
+  }
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(.open) {
+  --pfe-toast--Right: 50px;
+}
+
+:host(:not(.open)){
+  --pfe-toast--Right: calc(-1 * (#{pfe-local(MaxWidth)} + #{pfe-local(Right--offset)}));
+}
+
+::slotted(:first-child) {
+  margin-top: 0;
+}
+
+::slotted(button) {
+  background: none;
+  border-radius: 0;
+  display: inline-block;
+  margin-bottom: 0;
+  margin-right: 1rem;
+  padding: .5rem .75rem;
+  text-align: center;
+}
+
+.pfe-toast {
+  &__container {
+    // flex child props
+    align-self: flex-start;
+    flex: 1 1 auto;
+
+    // flex parent props
+    align-items: flex-start;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+
+    background-color: pfe-local($cssvar: BackgroundColor, $region: container);
```

These functions should be interpolated to ensure they resolve correctly:

```suggestion
    background-color: #{pfe-local($cssvar: BackgroundColor, $region: container)};
```